wirepas_mqtt_library/wirepas_network_interface.py

```diff
@@ -197,6 +197,16 @@ def _on_disconnect_callback(self, client, userdata, rc):
             self._task_queue.add_task(self._connection_cb, False, self.ConnectionErrorCode.from_broker_connack(rc))
 
     def _on_status_gateway_received(self, client, userdata, message):
+        if message.payload.__len__() == 0:
+            # Remove gateway if in our list
+            try:
+                del self._gateways[TopicParser.parse_status_topic(message.topic)]
+            except KeyError:
+                print(TopicParser.parse_status_topic(message.topic), "not in list")
```

Comment:
Instead of print(), maybe use the logger? Libraries should not generate output on stdout or stderr.